internal/app/apid/main.go

```diff
@@ -46,14 +54,7 @@ func Main() {
 
 	flag.Parse()
 
-	go func() {
-		debugLogFunc := func(msg string) {
-			log.Print(msg)
-		}
-		if lErr := debug.ListenAndServe(context.TODO(), debugAddr, debugLogFunc); lErr != nil {
-			log.Fatalf("failed to start debug server: %s", lErr)
-		}
-	}()
+	go runDebugServer(context.TODO())
```

Comment:
Extracted to reduce cyclomatic complexity without slapping `//nolint:gocyclo` on it